src/models/person/personsok.ts

```diff
@@ -10,6 +10,13 @@ export interface PersonsokResponse {
     status?: Kodeverk;
     ident: NorskIdent;
     brukerinfo: Brukerinfo | null;
+    utenlandskID?: UtenlandskID | null;
+    request?: PersonsokRequest;
+}
+
+export interface UtenlandskID {
+    utenlandskID: string | null;
+    type: Kodeverk;
```

Comment:
Denne er nok forbeholdt tilrettelagt kommunikasjon? Er det samme kodeverkstype kan vi lage en egen interface for det? :)

Reply:
Hva vil tilrettelagt kommunikasjon si?